src/test/java/hudson/plugins/promoted_builds/conditions/inheritance/SelfPromotionInheritanceTest.java

```diff
@@ -179,9 +180,15 @@ public void testPromotionEnvironmentShouldIncludeTargetParameters() throws Excep
         inheritanceProjectPair.getBase().addProperty(promotion);
 
         inheritanceProjectPair.getBase().addProperty(new ParametersDefinitionProperty(new StringParameterDefinition(paramName, "")));
+        inheritanceProjectPair.getDerived().addProperty(new ParametersDefinitionProperty(new StringParameterDefinition(paramName, "")));
```

Comment:
Smells like a bug in the in Inheritance Project plugin.
Makes sense to add TODO and to refer https://issues.jenkins-ci.org/browse/JENKINS-34831